seedData.js

```diff
@@ -0,0 +1,119 @@
+module.exports = {
```

Comment:
I think if you turn this into an array of objects, your randomProduct function in seed.js can be refactored to be much more eloquent.